django/bossspatialdb/views.py

```diff
@@ -488,3 +488,117 @@ def delete(self, request, collection, experiment, channel):
         channel_obj.save()
 
         return HttpResponse(status=204)
+
+class CutoutToBlack(APIView):
+    """
+    View to handle spatial cutouts by providing all datamodel fields
+
+    * Requires authentication.
+    """
+    # Set Parser and Renderer
+    parser_classes = (BloscParser, BloscPythonParser, NpygzParser, BrowsableAPIRenderer)
+    renderer_classes = (BloscRenderer, BloscPythonRenderer, NpygzRenderer, JpegRenderer,
+                        JSONRenderer, BrowsableAPIRenderer)
+
+    def __init__(self):
+        super().__init__()
+        self.data_type = None
+        self.bit_depth = None
+    
+    def put(self, request, collection, experiment, channel, resolution, x_range, y_range, z_range, t_range=None):
```

Comment:
Why was `PUT` selected instead of `DELETE`?

Reply:
Hmm I think `DELETE` would be more intuitive to understand for the user. However just from following the convention since this isn't deleting the resource but instead modifying it to contain only 0s in certain sections, I thought it made sense for it to be a `PUT`. @sandyhider @movestill  Any thoughts?

Reply:
Yeah that was the thought.

Reply:
Sounds logical